.github/workflows/ci.yaml

```diff
@@ -1,30 +1,44 @@
 name: CI
+
 on:
     pull_request:
-        branches:
-            - '**'
+        branches: ['*']
+    push:
+        branches: ['main']
+    merge_group:
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+env:
+    FORCE_COLOR: 3
```

Comment:
`FORCE_COLOR` ?ㅁ?

Reply:
https://force-color.org/#:~:text=Command%2Dline%20software%20which%20outputs%20colored%20text%20should,is%20present%20and%20not%20an%20empty%20string.

Reply:
for github actions terminals